File: CorejavaExample/Day02Assignment/src/com/sonata/Student.java
package com.sonata;

public class Student
{
	int stdId;
	String stdName;
	double stdMarks;
	Student(int n, String name, double b)
	{
		this.stdId= n;
		this.stdName= name;
		this.stdMarks=b;
		}
		Student()
		{
			this.stdId= 21259;
			this.stdName= "M G VIVEK";
			this.stdMarks=56;
			
		}
	
	public void Display()
	{
		System.out.println("student id: " + stdId);
		System.out.println("student Name: " + stdName);
		System.out.println("student Total marks out of 100 : " + stdMarks);
	}
	public static void main(String[] args) 
	{
		
		Student s1 =new Student();
		System.out.println("Details");
		s1.Display();
		Student s2 = new Student(21256,"KISHORE",78);
		s2.Display();
}


}

File: Typescript_2021/Student.ts
export class Student {
    stdId:number= 21259;
    stdName:string="vivek";
    stdMarks:number = 75
    constructor (i:number, n:string, s:number){
        this.stdId = i;
        this.stdName= n;
        this.stdMarks = s;
    }
    display(){
        console.log(this.stdId);
        console.log(this.stdName);
        console.log(this.stdMarks);
    }
}

var e1 = new Student(21250, "bolu",66);
e1.display();